docs/inputs.md

```diff
@@ -8,5 +8,7 @@
 | dns\_managed\_zone | The name of the managed DNS zone in which the application will be accessible. | `string` | n/a | yes |
 | dns\_managed\_zone\_dns\_name | The fully qualified DNS name of the managed zone set by var.dns\_managed\_zone. | `string` | n/a | yes |
 | labels | A collection of labels which will be applied to resources. | `map(string)` | `{}` | no |
+| secondaries\_max\_instances | The maximum count of compute instances to which the secondaries may scale. The default value is deferred to the secondaries submodule. | `number` | n/a | yes |
```

Comment:
sorry about the pedantism here but `deferred to` in this context seems slightly off as to what you are trying to convey.
if I'm missing the nuance here def let me know, but I'm thinking something like either 

```suggestion
| secondaries\_max\_instances | The maximum count of compute instances to which the secondaries may scale. The default value is derived from the secondaries submodule. | `number` | n/a | yes |
```

or

```suggestion
| secondaries\_max\_instances | The maximum count of compute instances to which the secondaries may scale. The default value is inferred from the secondaries submodule. | `number` | n/a | yes |
```

could represent this in a way of defining not a putting off of a decision to the secondaries submodule but a value that is a conclusion/result of the secondaries module?

Reply:
Nice catch! I forgot to update the description after trying to use `null` as the default here. :blush:

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/71/commits/bcab090be0fab982749532272b16b9d4ed4addfb.